### ISSUE TITLE
nn.functional.sigmoid is deprecated

### ISSUE THREAD

**Issue by panovr** [sdsubhajitdas/Brain-Tumor-Segmentation#14]
This [line ](https://github.com/sdsubhajitdas/Brain-Tumor-Segmentation/blob/master/bts/model.py#L129)
`output = F.sigmoid(self.conv10(conv9))
`
cause 
`UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")`

It should be:
`output = torch.sigmoid(self.conv10(conv9))`
